server.js

```diff
@@ -30,6 +39,24 @@ app.post("/api/v1/favorites", (request, response) => {
     return response.status(200).json(app.locals.usersFavorites);
 });
 
+app.get("/api/v1/comments", (request, response) => {
+	response.status(200).json(app.locals.comments);
+});
+
+app.get("/api/v1/comments/:movie_id", (request, response) => {
+	const movieID = parseInt(request.params.movie_id);
+	const commentMovieKeys = Object.values(app.locals.comments.comments);
```

Comment:
maybe change this to something like app.locals.allComments.movieComments ? seems a bit confusing saying .comments.comments